binnacle/lib/ui/CompassHeadingUI.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+
+import 'package:sos/utils/Math.dart';
+import 'ArrowPainter.dart';
+
+class CompassHeadingUI extends StatelessWidget {
+  CompassHeadingUI({Key key, this.heading, this.color}) : super(key: key);
+
+  // There's an issue with the compass stack. The sizes are different for
```

Comment:
This is fine for now, but we should put a todo here saying this needs to be fixed eventually